crawl-ref/source/player-equip.cc

```diff
@@ -604,9 +583,9 @@ static void _equip_weapon_effect(item_def& item, bool showMsgs, bool unmeld)
             switch (special)
             {
             case SPWPN_DISTORTION:
-                if (!was_known)
+                if (item.props["newly_identified"].get_bool())
```

Comment:
This doesn't really make any sense - there's no "surprise, you accidentally wielded a distortion weapon!". We should just toss this whole branch.